projects/aca-shared/src/lib/components/toolbar/toolbar-button/toolbar-button.component.ts

```diff
@@ -45,6 +45,20 @@ export enum ToolbarButtonType {
   selector: 'app-toolbar-button',
   templateUrl: './toolbar-button.component.html',
   encapsulation: ViewEncapsulation.None,
+  styles: [
+    `
+      .app-toolbar-button button:focus {
+        background-color: #1f74db3d;
+        border: 2px solid #1f74db;
```

Comment:
this does not seem right. Here we need to add a theme color.

Reply:
Removed this code as per the design changes